internal/windevice/devicequery.go

```diff
@@ -0,0 +1,120 @@
+package windevice
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf16"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+	"github.com/Microsoft/hcsshim/internal/winapi"
+	"github.com/pkg/errors"
+)
+
+const (
+	_CM_GETIDLIST_FILTER_BUSRELATIONS = uint32(0x00000020)
+
+	_CM_LOCATE_DEVNODE_NORMAL = uint32(0x00000000)
+
+	_DEVPROP_TYPE_STRING        = uint32(0x00000012)
+	_DEVPROP_TYPEMOD_LIST       = uint32(0x00002000)
+	_DEVPROP_TYPE_STRING_LIST   = uint32(_DEVPROP_TYPE_STRING | _DEVPROP_TYPEMOD_LIST)
```

Comment:
nit but why not just have the type on the left? 
```go
         _CM_GETIDLIST_FILTER_BUSRELATIONS uint32 = 0x00000020
	_CM_LOCATE_DEVNODE_NORMAL uint32         = 0x00000000
	_DEVPROP_TYPE_STRING  uint32                       = 0x00000012
	_DEVPROP_TYPEMOD_LIST  uint32                    = 0x00002000
	_DEVPROP_TYPE_STRING_LIST uint32                = _DEVPROP_TYPE_STRING | _DEVPROP_TYPEMOD_LIST
```

Reply:
Well that didn't format well haha You get the idea though.

Reply:
didn't think of it :) I'll update